package.json

```
@@ -1,46 +1,29 @@
-    "gitbook-plugin-addcssjs": "^1.0.2",
-    "gitbook-plugin-medium-zoom": "^1.1.0",
-    "gitbook-plugin-search-plus": "^1.0.3",
-    "gitbook-plugin-theme-default": "^1.0.7",
-    "markdownlint-cli": "^0.22.0",
-    "run-script-os": "^1.1.1",
-    "spellchecker-cli": "^4.2.0"
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "authors": {
+    "name": "",
+    "email": ""
   },
+  "repository": "/OpenCore-Desktop-Guide",
   "scripts": {
-    "build": "gitbook install && gitbook build",
-    "fix-lint": "run-script-os",
-    "fix-lint:default": "echo Attempting to fix lint... && markdownlint '**/*.md' -f && echo Fixed successfully, please commit. || echo Fix failed!",
-    "fix-lint:win32": "echo Attempting to fix lint... && markdownlint **/*.md -f && echo Fixed successfully, please commit. || echo Fix failed!",
-    "lint": "run-script-os",
-    "lint:default": "echo Linting... && markdownlint '**/*.md' && echo Lint passed. || echo Lint failed! Please review and fix errors.",
-    "lint:win32": "echo Linting... && markdownlint **/*.md && echo Lint passed. || echo Lint failed! Please review and fix errors.",
-    "lint-ci": "run-script-os",
-    "lint-ci:default": "echo Linting... && markdownlint '**/*.md' && echo Lint passed. || (echo Lint failed, attempting fix... && markdownlint '**/*.md' -f && echo Fix generated successfully. Please apply the following diff using git apply && git diff || echo Fix failed!)",
-    "lint-ci:win32": "echo Linting... && markdownlint **/*.md && echo Lint passed. || (echo Lint failed, attempting fix... && markdownlint **/*.md -f && echo Fix generated successfully. Please apply the following diff using git apply && git diff || echo Fix failed!)",
-    "serve": "gitbook install && gitbook serve",
-    "sort-dict": "node ./scripts/sortDict.js",
-    "spellcheck": "run-script-os",
-    "spellcheck:default": "spellchecker --plugins spell indefinite-article repeated-words syntax-urls --dictionaries dictionary/dictionary.txt dictionary/opencorekeys.txt --files '**/*.md'  && echo Spellcheck passed. || echo Spellcheck failed! Please review and fix errors/add words to dictionary as needed.",
-    "spellcheck:win32": "spellchecker --plugins spell indefinite-article repeated-words syntax-urls --dictionaries dictionary/dictionary.txt dictionary/opencorekeys.txt --files **/*.md  && echo Spellcheck passed. || echo Spellcheck failed! Please review and fix errors/add words to dictionary as needed.",
-    "test": "run-script-os",
-    "test:default": "npm run lint --silent; npm run spellcheck --silent",
-    "test:win32": "npm run lint --silent & npm run spellcheck --silent"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/dortania/OpenCore-Desktop-Guide.git"
+    "dev": "vuepress dev",
+    "build": "vuepress build"
   },
-  "author": "Dortania",
-  "license": "UNLICENSED",
-  "bugs": {
-    "url": "https://github.com/dortania/OpenCore-Desktop-Guide/issues"
+  "license": "MIT",
```

```suggestion
  "license": "UNLICENSED",
```
Need to import the license from laptop guide